app/code/Magento/Customer/Block/Form/Register.php

```diff
@@ -228,4 +231,19 @@ public function getRequiredCharacterClassesNumber()
     {
         return $this->_scopeConfig->getValue(AccountManagement::XML_PATH_REQUIRED_CHARACTER_CLASSES_NUMBER);
     }
+
+    /**
+     * Returns whether to show address fields or not
+     *
+     * @return bool
+     */
+    public function isShowAddressFields(): bool
```

Comment:
Could you move this method to the separate viewmodel class? It will prevent SVC failures.

More info there https://devdocs.magento.com/guides/v2.4/extension-dev-guide/view-models.html

Reply:
Ah. No, it won't help to resolve SVC failure as it also added a new configuration field.